modules/core/exported/channel.go

```diff
@@ -34,6 +34,16 @@ type PacketI interface {
 // Acknowledgement defines the interface used to return
 // acknowledgements in the OnRecvPacket callback.
 type Acknowledgement interface {
+	// Success determines if the IBC application state should be persisted when handling `RecvPacket`.
+	// During `OnRecvPacket` IBC application callback execution, all state changes are held in a cache store and committed if:
+	// - the acknowledgement.Success() returns true
+	// - a nil acknowledgement is returned (asynchronous acknowledgements)
+	// 
```

Comment:
wildest suggestion ever but this is what's causing the linter to fail (extra space)

```suggestion
	//
```